tests/test_app.py

```diff
@@ -255,6 +360,10 @@ def test_missing_nav(self):
         """
 
         response_1 = self.client_no_nav.get("/")
+
+        # The origin index isn't the "index" defined in client_no_nav
```

Comment:
Does this correspond to the id? Which is 42 on no-nav?

Reply:
Yep